content/components/action-bar.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Action bar
```

Comment:
Reminder to add ActionBar to the component index 😄 https://primer.style/design/components